zeapp/desktop/src/main/kotlin/de/berlindroid/zekompanion/desktop/ZeBadgeKompanion.kt

```diff
@@ -71,14 +72,55 @@ private fun sendResult(result: Result<Int>) {
     }
 }
 
+private fun State.EditImage.toBufferedImage(): BufferedImage {
+    val img = BufferedImage(296, 128, BufferedImage.TYPE_INT_RGB)
+    val g = img.createGraphics()
+
+    val badge = ComposePanel()
+    badge.size = Dimension(296 * 2, 128 * 2)
+    badge.setContent {
+        ImageEditorBadge(
+            image,
+            File(configFileName).readText().fromJSONtoConfig(),
+        )
+    }
+
+    return with(JFrame("badge")) {
```

Comment:
![jf](https://github.com/gdg-berlin-android/ZeBadge/assets/1771914/cafadb53-0aaa-4ab8-ac9b-1298093f3ab8)

Reply:
And I do love me some AI images ... :smirk: